strings.json

```
@@ -4,5 +4,11 @@
     "online": "{{count}} online",
     "members": "{{count}} Membros",
     "button": "Juntar-se"
+  },
+  "lt": {
+    "header": "Tu buvai pakviesta(s) prisijungti prie serverio",
```

```suggestion
    "header": "Tu buvai pakviestas prisijungti prie serverio",
```

I know, I know. But they should match Discord's strings.